src/app/dashboard/components/item-request/item-request.component.ts

```diff
@@ -17,20 +19,23 @@ import { Status } from 'src/app/core/constants/enums';
 })
 export class ItemRequestComponent implements OnInit, OnDestroy {
   @Input() vm: IItemDetailState;
-  @Output() createNote = new EventEmitter<Pick<ICreateOrderNoteInput, 'noteBody' | 'itemId'>>();
-  @Output() updateItem = new EventEmitter<{ orderUpdate: Agreement; updates: Partial<OrderChangeInput> }>();
+  @Output() createNote = new EventEmitter<Pick<ICreateOrderNoteInput, 'noteBody' | 'itemId' | 'imageUrl'>>(); 
+  @Output() updateItem = new EventEmitter<{ orderUpdate: Agreement, updates: Partial<OrderChangeInput> }>();
 
   status = Status; // enum binding to use in view template
-
+  userProfile: UserProfile;
   stop$ = new Subject();
-
+  showImageArea:boolean = false;
+  imagename: string = '';
   currentNoteVal: string;
   orderNoteFC: FormControl = new FormControl('');
   orderNoteFC$: Observable<string>;
 
-  constructor() {}
+  constructor(private userService: UserService) { }
 
   ngOnInit() {
+    this.userProfile = this.userService.getCurrentUserProfile();
```

Comment:
These components are built with push based vs. pull based architecture and the preference would be to have the service fetch userprofile and expose it to child components via the state observable as opposed to the child component fetching it.